docs/pages/state-query/dozer.mdx

```diff
@@ -0,0 +1,573 @@
+  ],
+  [
+    ["id", "description"],
+    ["0xb15fd0e41ab0bb6eb992e0a3d4f30fce6ee24a5fc9c30f725fdfc96d9d16ed95", "Do the dishes"],
+    ["0xb81d5036d0b62e0f2536635cbd5d7cec1d1f0706c0c6c1a9fa74293d7b0888eb", "Take out the trash"]
+  ]
+]
+```
+
+</details>
+
+### Sorted results
+
+If you want to control the order in which you get results, you can use an `ORDER BY` clause.
+
+```json filename="query.json" copy
+[
+  {
+    "address": "0x95F5d049B014114E2fEeB5d8d994358Ce4FFd06e",
+    "query": "SELECT description, createdAt FROM app__Tasks ORDER BY createdAt"
+  }
+]
+```
+
+Note that the sort field(s) need to be part of the selected columns.
+
+<details>
+
+<summary>Results</summary>
+
+```json
+[
+  ["description", "createdat"],
+  ["Walk the dog", "1723495628"],
+  ["Take out the trash", "1723495640"],
+  ["Do the dishes", "1723495642"],
+  ["Test", "1723495964"],
+  ["Test from a different account", "1723576522"],
+  ["Another test", "1723576522"],
+  ["Yet another test", "1723646440"]
+]
+```
+
+</details>
+
+### Multiple tables
+
+You can join multiple tables, using the same syntax SQL uses.
+
+```json filename="query.json" copy
+[
+  {
+    "address": "0x95F5d049B014114E2fEeB5d8d994358Ce4FFd06e",
+    "query": "SELECT app__Creator.id, description, taskCreator FROM app__Tasks, app__Creator WHERE app__Creator.id=app__Tasks.id"
+  }
+]
+```
+
+<details>
+
+<summary>Results</summary>
+
+```json
+[
+  ["id", "description", "taskcreator"],
+  [
+    "0x3e0a112aadc5e02927fb4a91649bea565fd1baa1175aae4cb4957d6348f165cf",
+    "Test",
+    "0x735b2f2c662ebedffa94027a7196f0559f7f18a4"
+  ],
+  [
+    "0x727d7bfe00b6db638c69595059dc10e21c52a7912d090905a7c7dc8659efd3b8",
+    "Test from a different account",
+    "0x428b1853e5ec29d35c84a218ec5170efc7621b58"
+  ],
+  [
+    "0xb15fd0e41ab0bb6eb992e0a3d4f30fce6ee24a5fc9c30f725fdfc96d9d16ed95",
+    "Do the dishes",
+    "0x8225d72f2c39f3729d7f3fc03c6aa8731eaeef48"
+  ],
+  [
+    "0xb81d5036d0b62e0f2536635cbd5d7cec1d1f0706c0c6c1a9fa74293d7b0888eb",
+    "Take out the trash",
+    "0x8225d72f2c39f3729d7f3fc03c6aa8731eaeef48"
+  ],
+  [
+    "0xd43394ecf79077f65cd83b534dd44d3b4e9e2aa553e95aafecd14b8529543cda",
+    "Another test",
+    "0x428b1853e5ec29d35c84a218ec5170efc7621b58"
+  ]
+]
+```
+
+</details>
+
+### Grouping results
+
+You can use `GROUP BY` to identify different groups.
+For example, this query gets you the different task creators.
+
+```json filename="query.json" copy
+[
+  {
+    "address": "0x95F5d049B014114E2fEeB5d8d994358Ce4FFd06e",
+    "query": "SELECT taskCreator FROM app__Creator GROUP BY taskCreator"
+  }
+]
+```
+
+<details>
+
+<summary>Results</summary>
+
+```json
+[
+  ["taskcreator"],
+  ["0x428b1853e5ec29d35c84a218ec5170efc7621b58"],
+  ["0x735b2f2c662ebedffa94027a7196f0559f7f18a4"],
+  ["0x8225d72f2c39f3729d7f3fc03c6aa8731eaeef48"]
+]
+```
+
+</details>
+
+## Using dozer from a MUD client
```

Comment:
for the typescript section i think it would be useful to have the part about "how to modify a MUD template to hydrate from dozer", but additionally before that i would put a simpler example with just a single typescript file:

```ts
const config = defineStore({ ... });
const result = fetchRecordsSql({ 
  dozerUrl: ...,
  storeAddress: ...,
  queries: [
    selectFrom(...)
  ]
 });
```